src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -82,6 +82,28 @@ CHIP_ERROR SystemCommands::Start(const char * identity, const chip::app::Cluster
         builder.Add(value.minCommissioningTimeout.Value());
     }
 
+    // OTA provider specific arguments
+    if (value.filepath.HasValue())
+    {
+        VerifyOrReturnError(value.filepath.Value().size() < 128, CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Do we want to add a #define instead of hard-coding 128?

Reply:
Yeah, I was just cop/paste from above but it's used enough times that I should define something for it.